src/Server/Utils/Strategies/RegexResponseStrategy.php

```diff
@@ -70,6 +74,29 @@ private function getResponseWithBody(
 
         return $httpResponse;
     }
+    
+    /**
+     * @param Expectation            $expectation
+     * @param ResponseInterface      $httpResponse
+     * @param ServerRequestInterface $httpRequest
+     *
+     * @return \Psr\Http\Message\ResponseInterface
+     */
+    protected function getResponseWithReplacedHeaders(
```

Comment:
I'm not a huge fan of this name, willing to take suggestions

Reply:
I think the name is explanatory, since we are in the RegexResponseStrategy class.